### ISSUE TITLE
Test failure

### ISSUE THREAD

**Issue by orhun** [josueBarretogit/get_blessed_rs#4]
Hey!

I got the following test failure while trying to package `get_blessed_rs` for the AUR:

```
running 1 test
test content_parser::test::general_table_has_expected_data ... FAILED

failures:

---- content_parser::test::general_table_has_expected_data stdout ----
thread 'content_parser::test::general_table_has_expected_data' panicked at src/content_parser.rs:14:9:
not yet implemented
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace


failures:
    content_parser::test::general_table_has_expected_data

test result: FAILED. 0 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

error: test failed, to rerun pass `--bin get-blessed`
```

I realized it is just an empty test with a `todo!()` - it might be a good idea to remove it or add `#[ignore]` until it is functional.

**Comment by josueBarretogit** [comment]
Hi yes at the moment there is no test I don´t know how to do proper tests

**Comment by orhun** [comment]
I took a look at the codebase, maybe you can create a new `ContentParser`, fetch the contents and assert the result there. But I'm not too sure.

**Comment by josueBarretogit** [comment]
Closed #11
